charmcraft/commands/build.py

```diff
@@ -164,13 +165,33 @@ def build_charm(self, bases_config: BasesConfiguration) -> str:
             shutil.rmtree(str(self.buildpath))
         self.buildpath.mkdir()
 
-        create_manifest(self.buildpath, self.config.project.started_at, bases_config)
-
         linked_entrypoint = self.handle_generic_paths()
         self.handle_dispatcher(linked_entrypoint)
         self.handle_dependencies()
-        zipname = self.handle_package(bases_config)
 
+        linting_results = []
+        # run linters, present them to the user according to their type, and decide if go on
```

Comment:
"decide if go on" is missing words, I would inverse the statement to not use so many words
```suggestion
        # run linters, present them to the user according to their type, and fail if necessary
```

or maybe just remove as there is not decision being made yet.

Reply:
thanks, improved

Reply:
btw, left it, as I put the XXX below...